src/test/java/org/springframework/samples/petclinic/PostgresIntegrationTests.java

```diff
@@ -125,11 +128,16 @@ public void printProperties() {
 					assertNotNull(sourceProperty.toString(), "source property toString() returned null.");
 
 					String value = sourceProperty.toString();
+
 					if (resolved.equals(value)) {
-						log.info(name + "=" + resolved);
-					}
-					else {
-						log.info(name + "=" + value + " OVERRIDDEN to " + resolved);
+						if (log.isInfoEnabled()) {
+							log.info(name + "=" + resolved);
+						}
+						else {
```

Comment:
I think the braces fall in the wrong place here. Surely we can just put the `isInfoEnabled()` check around the whole of the code that used to be here?